README.md

```diff
@@ -27,3 +27,7 @@ about the current recursive call.
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. You do not need to prove that the invariant is correct. Add
 your answer to this markdown file.
+
+An invariant for fib(n) would be the two previous numbers in the sequence, fib(n-1) and fib(n-2). Each recursive call computes fib(n) by adding these two values. This recursive call starts with fib(0) and fib(1) in the array which satifies the base case.
```

Comment:
This is not an invariant. "the two previous numbers in the sequence, fib(n-1) and fib(n-2)" cannot be true or false. Do you mean that those numbers should have particular values?